src/internal/js/mod.rs

```diff
@@ -307,10 +313,26 @@ impl JsString {
         }
     }
 
+    pub fn ucs2_value(self) -> Vec<u16> {
```

Comment:
Ah true. But then, what's the benefit of doings this - it still doesn't guarantee that data is not being transformed on the fly, because, as far as I remember at least, V8 can use also UTF-8 and Latin-1 as internal representations for the string, which would mean that sometimes this will introduce transforms to UCS-2 which weren't done previously.

Can't we instead construct strings in V8 directly from UTF-8 representation that Rust provides? (and then cases like Latin-1 and UTF-8 -> Rust usually won't require any conversion, just copies)